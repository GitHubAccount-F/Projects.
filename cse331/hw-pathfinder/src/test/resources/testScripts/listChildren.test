#Tests that ListChildren works with nodes of type double

#Creates graphs
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3

#Adds nodes
AddNode graph1 one
AddNode graph1 two

AddNode graph2 one
AddNode graph2 two

AddNode graph3 one
AddNode graph3 two
AddNode graph3 three

#Adds edges
AddEdge graph1 one two 5

AddEdge graph2 one two 5
AddEdge graph2 one two 4

AddEdge graph3 one two 5
AddEdge graph3 one two 7.3509
AddEdge graph3 one two 44.901
AddEdge graph3 one three 70

#List the child nodes
ListChildren graph1 one
ListChildren graph2 one
ListChildren graph3 one