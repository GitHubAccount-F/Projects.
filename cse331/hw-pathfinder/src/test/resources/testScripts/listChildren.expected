#Tests that ListChildren works with nodes of type double

#Creates graphs
created graph graph1
created graph graph2
created graph graph3

#Adds nodes
added node one to graph1
added node two to graph1

added node one to graph2
added node two to graph2

added node one to graph3
added node two to graph3
added node three to graph3

#Adds edges
added edge 5.000 from one to two in graph1

added edge 5.000 from one to two in graph2
added edge 4.000 from one to two in graph2

added edge 5.000 from one to two in graph3
added edge 7.351 from one to two in graph3
added edge 44.901 from one to two in graph3
added edge 70.000 from one to three in graph3

#List the child nodes
the children of one in graph1 are: two(5.000)
the children of one in graph2 are: two(4.000) two(5.000)
the children of one in graph3 are: three(70.000) two(5.000) two(7.351) two(44.901)